### DISCUSSION TITLE
thank you

### DISCUSSION THREAD

**Discussion by ryanhs** [jreyesr/insomnia-plugin-batch-requests#16] [Show and tell]
i just want to say thank you :D
:+1: 

since postman have 25 limit, not good for hobbist

this plugin you made is what best alternative available now :+1:

**Comment by jreyesr** [comment]
Hello! Thanks for the kind words, they're much appreciated.

I didn't know that Postman limited users to 25 runs per month. That really sucks if you just want to run some requests against a CSV file. Oh well. Glad the plugin can help!

If you think of any features that may be helpful, feel free to suggest them. [Several recent releases](https://github.com/jreyesr/insomnia-plugin-batch-requests/releases) (e.g. the ability to read output from headers and the status code, and most recently the ability to send files that vary across requests) have been driven by people asking.
